ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/AUTOVPN_TESTS.yml

```diff
@@ -45,3 +53,78 @@ wan_rr:
       id: 1
     - name: autovpn-rr2
       id: 2
+
+wan_path_groups:
+  - name: MPLS
+    ipsec: False
+    id: 100
+  - name: INET
+    id: 101
+  - name: LTE
+    id: 102
+
+wan_carriers:
+  - name: Comcast
+    path_group: INET
+  - name: ATT
+    path_group: INET
+
+l3_edge:
+  l3_interfaces:
+    - node: autovpn-edge
+      interface: Ethernet1
+      wan_carrier: Comcast
+      wan_circuit_id: 666
+      ip: dhcp
+      set_default_route: true
+    - node: autovpn-rr1
+      interface: Ethernet1
+      wan_carrier: ATT
+      wan_circuit_id: 777
+      ip: dhcp
+      set_default_route: true
+    - node: autovpn-rr2
+      interface: Ethernet1
+      wan_carrier: ATT
+      wan_circuit_id: 888
+      ip: 10.8.8.8/31
+      set_default_route: true
+      peer_ip: 10.8.8.9/31
+
+tenants:
+  - name: TenantA
```

Comment:
I think the word Tenant is something we decided to not use. Just VRFs everywhere.

Reply:
This is an existing AVD construct, so not something built for this purpose alone.

Reply:
Thanks. Guillaume have more context. I mistook it to mean something else.